lib/lotus/loader.rb

```diff
@@ -60,6 +61,7 @@ def _configure_controller_framework!
             cookies config.cookies.default_options
           end
           prepare { include Lotus::Action::Session } if config.sessions.enabled?
+          prepare { include Lotus::Action::RoutingHelpers }
```

Comment:
@AlfonsoUceda what do you think about including this helper in view framework too?